src/main.rs

```diff
@@ -113,6 +114,13 @@ async fn main() {
         configuration.ips_config.peer_file_path = arg_conf.ips_file;
     }
 
+    // Check if user error setting optional filter type
+    if arg_conf.filter_type.is_some() && arg_conf.filter_type.unwrap() == NetworkType::Invalid {
+        panic!("Invalid network type for filter. Should be Zcash or Ripple.")
```

Comment:
Hopefully, when there will be new network added we won't forget to update this message :) Can be connected with previous review note and end up suggesting user reading the Readme to get possible values.

```suggestion
        panic!("Invalid network type for filter. Check Readme for possible values.")
```

Reply:
done!